zones/stardewmodding.wiki

```diff
@@ -0,0 +1,32 @@
+$TTL 300
+$ORIGIN stardewmodding.wiki.
+
+@		SOA ns1.miraheze.org. hostmaster.miraheze.org. (
+		20210416000001	; serial
```

Comment:
This needs to be updated to today, so '20210526' and then the rest stays the same.

Reply:
Done.